src/Services/PhoneHomeService.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Enjin\Platform\Services;
+
+use Enjin\Platform\Enums\Global\PlatformCache;
+use Enjin\Platform\Enums\TelemetrySource;
+use Enjin\Platform\Http\Controllers\PlatformController;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Str;
+use Throwable;
+
+class PhoneHomeService
+{
+    public function phone(TelemetrySource $source = TelemetrySource::REQUEST): void
+    {
+        if (!config('telemetry.enabled') || config('telemetry.enabled') === 'false') {
+            return;
+        }
+
+        try {
+            $uuid = Cache::rememberForever(
+                PlatformCache::TELEMETRY_UUID->value,
+                function () {
+                    if (!$uuid = DB::table('telemetry')->first()?->uuid) {
+                        DB::table('telemetry')->insert(['uuid' => $uuid = Str::uuid()->toString()]);
+                    }
+
+                    return $uuid;
+                }
```

Comment:
This can probably be moved to a `protected function getPlatformIdentifier(): string` to make it more readable.